### PR TITLE
migrate tao

### PR DESCRIPTION
**Pull request by EilidhRoss1**
have moved contents of tao.lifeitself.org (https://github.com/life-itself/tao) to lifeitself.org/tao (i.e. vault/tao in this repo). Have taken action based on: https://docs.google.com/spreadsheets/d/13RVMrZ02Rw6a1BtftIUO4v7R7rXt5_phQLR4rHa343M/edit#gid=453234015

### CODE REVIEW

**Comment by rufuspollock** [comment]
@EilidhRoss1 minor comment is that you have `.DS_Store` files in the commit. We'll need to remove them.

Also worth noting in PR summary that you have updated the spreadsheet with action status etc - that is the source of truth for steps to be taken here.

File: vault/notebook/five-mindfulness-trainings.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Five Mindfulness Trainings
+
+*   _From Thich Nhat Hanh_
```
Comment (rufuspollock) [inline vault/notebook/five-mindfulness-trainings.md:3]: Get this was a copy and paste ... and we probably don't want the bullet point here. These are examples of minor improvements we could make in the transfer over.

File: vault/notebook/integrity.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+## Integrity
```
Comment (rufuspollock) [inline vault/notebook/integrity.md:1]: This really should be a first level title i.e. `# Integrity`

File: vault/notebook/git.md [hunk 3]
```diff
@@ -0,0 +1,477 @@
+# An Introduction To Git
```
Comment (rufuspollock) [inline vault/notebook/git.md:1]: I don't think we want this - see comment in spreadsheet.

File: vault/tao/.github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+name: Continuous Deployment
```
Comment (rufuspollock) [inline vault/tao/.github/workflows/main.yml:1]: we don't want any of this stuff that isn't actual markdown.

File: vault/tao/hubs.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+# Hubs
```
Comment (rufuspollock) [inline vault/tao/hubs.md:1]: If you do have a `hubs` directory you probably want this file to be at `hubs/index.md`. both will work but that is better IMO.

**Review by rufuspollock — CHANGES_REQUESTED** [review]
Overall well done on the wrangling git and using obsidian (i assume you used that?).

Some changes needed based on my comments - and you have highlighted some things i hadn't made clear before e.g. around images etc.

**Comment by EilidhRoss1** [comment]
Thanks Rufus, am travelling today so my work day will be a bit disrupted but will try and get all this done today

**Comment by rufuspollock** [comment]
@EilidhRoss1 hi, no problem and as per my comment in the issue #182 i think best to start with a smaller pull request with say 5 top priority files (i think spreadsheet is in priority order) and then once that is good we move to rest. Does that sound good?

File: vault/notebook/five-mindfulness-trainings.md (hunk 1 above)
Comment (EilidhRoss1) [reply vault/notebook/five-mindfulness-trainings.md]: Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939

File: vault/notebook/integrity.md (hunk 2 above)
Comment (EilidhRoss1) [reply vault/notebook/integrity.md]: Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939

File: vault/notebook/git.md (hunk 3 above)
Comment (EilidhRoss1) [reply vault/notebook/git.md]: Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939

File: vault/tao/hubs.md (hunk 5 above)
Comment (EilidhRoss1) [reply vault/tao/hubs.md]: Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939

**Comment by rufuspollock** [comment]
@EilidhRoss1 there are a lot of conflicts here. I think you are going to need to close this pull request and start a new one. Is that possible?

**Comment by EilidhRoss1** [comment]
@rufuspollock sorry I'm still learning about all this, would that meaning starting again? I can do that, just want to make sure I'm understanding.

**Comment by rufuspollock** [comment]
I think starting again at this point is probably easiest

**Comment by EilidhRoss1** [comment]
Ok, will do 👍

**Comment by rufuspollock** [comment]
WONTFIX. Replace by other PRs.

**Pull request closed without merging**